stylix/mk-target.nix

```diff
@@ -0,0 +1,102 @@
+{
+  name,
+  humanName,
+  autoEnable ? true,
+  extraOptions ? { },
+  configElements ? [ ],
+  generalConfig ? null,
```

Comment:
And/or having some (brief) docs here to clarify what each arg relates to and/or how they should be used?